modular_chomp/code/modules/mob/living/simple_mob/subtypes/humanoid/eclipse/mechas.dm

```diff
@@ -317,162 +330,205 @@
 	downfour(target)
 	rightfour(target)
 	leftfour(target)
-	sleep(15)
 
 /mob/living/simple_mob/mechanical/mecha/eclipse/antipersonal_unit/proc/phase_two(atom/target) //SPIIIIN TO WIN
+	visible_message(span_danger("The [src] is preparing a deadly attack!"))
+	addtimer(CALLBACK(src, PROC_REF(spin_to_win), target), 2 SECOND, TIMER_DELETE_ME)
```

Comment:
I know it's just semantic, but 0.5, 2, etc should use SECONDS as plural form.

Reply:
Alright will alter in a moment

Reply:
thanks^^